collect_app/src/main/res/layout/geotrace_dialog.xml

```diff
@@ -15,39 +15,52 @@
             android:layout_marginLeft="10dp"
             android:layout_marginStart="10dp"
             android:layout_marginBottom="10dp"
-            android:onClick="updateRecordingMode"
-            android:checked="true"
             android:text="@string/manual_mode" />
 
         <RadioButton android:id="@+id/trace_automatic"
             android:layout_width="match_parent"
             android:layout_height="wrap_content"
-            android:clickable="true"
             android:layout_marginBottom="10dp"
             android:layout_marginLeft="10dp"
             android:layout_marginStart="10dp"
-            android:onClick="updateRecordingMode"
             android:text="@string/automatic_mode" />
 
     </RadioGroup>
 
-    <Spinner
-        android:id="@+id/trace_delay"
-        android:layout_width="match_parent"
-        android:layout_height="wrap_content"
-        android:entryValues="@array/delay_scale"
-        android:entries="@array/delay_scale"
-        android:visibility="gone"
+    <LinearLayout android:id="@+id/auto_options"
+                    android:orientation="vertical"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content">
+        <LinearLayout android:orientation="horizontal"
+                      android:layout_width="wrap_content"
+                      android:layout_height="wrap_content">
+          <TextView
+              android:layout_width="wrap_content"
+              android:layout_height="wrap_content"
+              android:paddingLeft="42dp"
+              android:paddingStart="42dp"
+              android:text="@string/recording_interval" />
+          <Spinner
+              android:id="@+id/auto_interval"
+              android:layout_width="match_parent"
+              android:layout_height="wrap_content" />
+        </LinearLayout>
```

Comment:
These two spinners are rather tightly displayed, can you add some padding around them?

Reply:
Yeah, that's a good idea.  Done.